vars/getVaultSecret.groovy

```diff
@@ -58,6 +60,20 @@ def readSecret(secret) {
   return props
 }
 
+def readSecretWrapperWithParams(role_id, secret_id, body) {
```

Comment:
If possible I'd say to use the `Maps args=[:]` this should help us to add parameters in the future without affecting the signature and therefore the consumers

Reply:
I update the signature to `(args, body)`.